src/bert/utils.py

```diff
@@ -12,6 +12,19 @@
 # huggingface implementation of BERT
 import pytorch_pretrained_bert
 
+SEP_ID = 104 # [SEP] for BERT models.
```

Comment:
Is this true for _all_ bert models? Might be safer to do a vocabulary lookup, since I'm not sure this is enforced anywhere else.